src/index.js

```diff
@@ -1,52 +1,168 @@
 import './style.css';
 import reIcon from './Refresh_icon.png';
-import tdIcon from './three_dots.png';
+import tdIcon from './trash-outline.svg';
 
 const inHead = document.querySelector('.div-heading');
 const ulList = document.getElementById('to-do-list');
+
+// adding imported image for the app heading
 const reicon = new Image();
 reicon.src = reIcon;
 reicon.className = 'reload-image';
 reicon.setAttribute('alt', 'reload-icon');
 inHead.appendChild(reicon);
 
-const tasks = [
-  {
-    description: 'Wash the dishes',
-    completed: true,
-    index: 5,
-  },
-  {
-    description: 'Complete to do list',
-    completed: true,
-    index: 4,
-  },
-  {
-    description: 'Make a Cake',
-    completed: true,
-    index: 3,
-  },
-  {
-    description: 'Shopping for the weekend',
-    completed: true,
-    index: 2,
-  },
-];
-
-const sortedTasks = [...tasks];
-const render = () => {
-  sortedTasks.sort((a, b) => a.index - b.index);
-  for (let i = 0; i < sortedTasks.length; i += 1) {
-    ulList.innerHTML += `
-      <li class='each-task'>
-        <div class='check-tag'>
-          <input type='checkbox' id='task-desc'>
-          <lable for='task-desc'>${sortedTasks[i].description}</label>
-        </div>
-        <img src=${tdIcon} class='three-dots' alt='three dots'/>
-      </li>
-    `;
+// variables for the outer div of the to-do-list heading
+const addlistdiv = document.querySelector('.list-adding');
+
+// Creating input element that takes the task and adds the task
+const addlist = document.createElement('input');
```

Comment:
- [ ] Please ensure that you add a button at the end of your input field to add tasks to enhance user experience. Currently adding a task to the list is somehow difficult because if I finish typing the task in the input field and I press enter key the task is not added however you don't have a button to be clicked to add the tasks to the list. :blush: 
  
  ![image](https://github.com/tan12082001/Project-To-Do-List/assets/92303891/5b421cec-c3cb-4c47-85b7-2d93d517d7e9)